documentation/book/assembly-storage.adoc

```diff
@@ -19,4 +19,14 @@ IMPORTANT: Once the Kafka cluster is deployed, the storage cannot be changed.
 
 include::ref-storage.adoc[leveloffset=+1]
 
+include::ref-storage-ephemeral.adoc[leveloffset=+1]
```

Comment:
If we add Ephemeral storage and Persistent storage as individual modules and leveloffset=+1, in the ToC it would look like 3.1.1.1. Storage, 3.1.1.2. Ephemeral storage, and 
3.1.1.3. Persistent storage and 3.1.1.3.1. Persistent Volume Claim naming.

If we see the logical order of the doc, Ephemeral storage and Persistent storage should be nested under Storage and not inline with Storage. I think this PR would move heading level 5 to level 4 but does not really solve the core issue.  I think we have to think of a different way to handle this doc architecture issue.

Reply:
That's not what I'm seeing when I do my local build.  It's all nested properly still.

Reply:
Sorry I was wrong, I see what you're saying, submitting another PR to fix.